src/Controller/Component/CrudComponent.php

```diff
@@ -275,7 +275,7 @@ public function execute(?string $controllerAction = null, array $args = []): Res
      */
     public function action(?string $name = null): BaseAction
     {
-        if (empty($name)) {
+        if ($name === null) {
```

Comment:
Not sure what the point of this change is

Reply:
These are changes of lines that psalm was complaining about, to make the pipeline pass. If not passing the pipeline is not a problem, the Existing psalm errors fixed commit can be removed.

Reply:
Looks to me like the coding standards passed. The failures look like test coverage

Reply:
Yes, now they pass, but without these changes, coding standards do not pass. I assume this was already in master, as I branched from master yesterday.